.travis.yml

```diff
@@ -13,20 +13,33 @@ python:
   - "3.4"
   - "3.5"
 install:
-  - sudo /etc/init.d/postgresql stop
-  - sudo apt-get -y remove --purge postgresql-9.1 postgresql-9.2 postgresql-9.3 postgresql-9.4
-  - sudo apt-get -y autoremove
-  - sudo apt-key adv --keyserver keys.gnupg.net --recv-keys 7FCC7D46ACCC4CF8
-  - sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ precise-pgdg main 9.5" >> /etc/apt/sources.list.d/postgresql.list'
-  - sudo apt-get update
-  - sudo apt-get -y install postgresql-9.5
-  - sudo /etc/init.d/postgresql stop
+  - |
+    if [[ $TEST_SUITE == "behave" ]]; then sudo bash -c '
+        /etc/init.d/postgresql stop
+        apt-get -y remove --purge postgresql-9.1 postgresql-9.2 postgresql-9.3 postgresql-9.4
+        apt-get -y autoremove
+        apt-key adv --keyserver keys.gnupg.net --recv-keys 7FCC7D46ACCC4CF8
+        echo "deb http://apt.postgresql.org/pub/repos/apt/ precise-pgdg main 9.5" >> /etc/apt/sources.list.d/postgresql.list
+        apt-get update
+        apt-get -y install postgresql-9.5
+        /etc/init.d/postgresql stop
+    '; fi
+  - |
+    if [[ $TEST_SUITE == "behave" ]]; then
```

Comment:
Would it be possible to merge those 2 blocks into one (the condition to check is the same)?